api/RTCIceCredentialType.json

```diff
@@ -64,40 +64,6 @@
             "deprecated": true
           }
         }
-      },
-      "token": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceCredentialType#token",
```

Comment:
It seems that this whole file should be removed. Maybe it was a dictionary?

Reply:
It seems like it was, especially since the link now redirects to `RTCIceServer.credentialType`.  I've opened #18496 to remove it.

Reply:
Thanks! Merged :)